src/main/webapp/resources/js/pages/projects/samples/linker/Linker.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { render } from "react-dom";
+import { Button, Modal, Typography } from "antd";
+import { CodeOutlined } from "@ant-design/icons";
+import { grey2, grey4, grey9 } from "../../../../styles/colors";
+import { getNGSLinkerCode } from "../../../../apis/linker/linker";
+import { SPACE_SM } from "../../../../styles/spacing";
+
+const { Paragraph, Text } = Typography;
+
+/**
+ * React component to display the ngs-linker command to a user based
+ * on the currently selected samples.
+ * @returns {*}
+ * @constructor
+ */
+function Linker() {
+  function handleSampleIds(e) {
+    // Post data to get linker command.
+    const { detail } = e;
+    getNGSLinkerCode(detail).then(({ data }) => {
+      Modal.success({
+        className: "t-linker-modal",
+        width: 500,
+        title: i18n("Linker.title"),
+        content: (
+          <>
+            <Paragraph>{i18n("Linker.details")}</Paragraph>
+            <Text type="secondary">
+              <span
+                dangerouslySetInnerHTML={{ __html: i18n("Linker.note") }}
+              />
+            </Text>
+            <Paragraph
+              style={{
```

Comment:
Since there are a lot of variables being set for this style maybe it would be a good idea to move it to the top of the file

Reply:
Updated in b8353892b38050da0d1384e16b7e34dd282b0211